builds/e2e/templates/stresstest-deploy-windows.yaml

```diff
@@ -39,6 +33,12 @@ steps:
       script: |
         #Requires -RunAsAdministrator
         $testName="Stress"
+        # schedule a task to stop iotedge service (terminate the test) in 4:30h
```

Comment:
I recently updated to move schedule task block after run test script.  Please checkout the latest version from master.

Reply:
I'm assuming this is only for our windows agents.